CONTRIBUTING.md

```diff
@@ -48,6 +48,16 @@ When submitting pull requests, make sure to do the following:
 - Remove trailing whitespace. Many editors will do this automatically.
 - Ensure any new files have [a trailing newline](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file)
 
+## Feature Stages
+
+Often, new features will need to go through experimental stages so that we can gather feedback and adjust as necessary.
+
+You can see this project's [feature stage documentation](https://agones.dev/docs/guides/feature-stages/) on the Agones
```

Comment:
Note - this will be a dead link until the website page is published. Probably ok, but the github changes go live immediately.

Reply:
True - it usually goes live within minutes of the commit going to master - so not something I'm deeply concerned about.

Reply:
I guess it's only relevant if we hide the feature stages until the next release (using a publish date in the future). If we are just going to push it live then it's not something to worry about.